packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__ternary_prop.snap

```diff
@@ -65,15 +65,15 @@ export const Cmp_component_4ryKJTOKjWE = ()=>{
         toggleSig
     ]);
     return /*#__PURE__*/ _jsxSorted("button", null, {
+        onClick$: handleClick$,
```

Comment:
This is now wrong. This function is a qrl with const captures, so it should be const.

Reply:
oops I misread this, it's still const just ordered differently